pkg/object/meshcontroller/spec/spec.go

```diff
@@ -211,21 +200,21 @@ type (
 
 	// Resilience is the spec of service resilience.
 	Resilience struct {
-		RateLimiter    *ratelimiter.Spec                `yaml:"rateLimiter" jsonschema:"omitempty"`
-		CircuitBreaker *resilience.CircuitBreakerPolicy `yaml:"circuitBreaker" jsonschema:"omitempty"`
-		Retryer        *resilience.RetryPolicy          `yaml:"retryer" jsonschema:"omitempty"`
-		TimeLimiter    *TimeLimiterPolicy               `yaml:"timeLimiter" jsonschema:"omitempty"`
+		RateLimiter    *ratelimiter.Rule     `yaml:"rateLimiter" jsonschema:"omitempty"`
+		CircuitBreaker *CircuitBreakerRule   `yaml:"circuitBreaker" jsonschema:"omitempty"`
+		Retry          *resilience.RetryRule `yaml:"retry" jsonschema:"omitempty"`
+		TimeLimiter    *TimeLimiterRule      `yaml:"timeLimiter" jsonschema:"omitempty"`
 	}
 
-	// TimeLimiterPolicy is the policy of TimeLimiter.
-	TimeLimiterPolicy struct {
-		Timeout string `yaml:"timeout" jsonschema:"required,format=duration"`
+	// CircuitBreakerRule is the spec of circuit breaker.
+	CircuitBreakerRule struct {
+		resilience.CircuitBreakerRule `yaml:",inline"`
+		FailureCodes                  []int `yaml:"failureCodes" jsonschema:"required,uniqueItems=true"`
```

Comment:
`failureCodes` is for both circuit breaker and retry to detect failures..

Reply:
Updated, and I also aligned the status layout for monitoring all objects by adding kubernetes-like namespace system.

Reply:
An partial example of status list:

```
default/StatusSyncController/restaurant-mesh:
  timestamp: 1658556750
default/easemesh-controller/delivery-mesh:
  timestamp: 1658556750
default/easemesh-controller/easemesh-control-plane-0:
  timestamp: 1658556750
default/easemesh-controller/easemesh-ingress-controller-5c4b75bc69-n884s:
  timestamp: 1658556750
default/easemesh-controller/order-mesh:
  timestamp: 1658556750
default/easemesh-controller/restaurant-mesh:
  timestamp: 1658556750
eg-traffic-easemesh-controller/ingresscontroller-server/easemesh-ingress-controller-5c4b75bc69-n884s:
  spec:
    autoCert: false
    caCertBase64: ""
    cacheSize: 0
    certBase64: ""
    certs: {}
    clientMaxBodySize: 0
    http3: false
    https: false
    keepAlive: true
```